learn-arai/app/components/icons/github.tsx

```diff
@@ -0,0 +1,12 @@
+import Image from "next/image";
+
+export const GitHubIcon = () => {
```

Comment:
Is there any reasons to use arrow function instead of regular function?
If there is only one export, consider default export this function